grails-app/controllers/org/pih/warehouse/api/ApiController.groovy

```diff
@@ -72,8 +73,26 @@ class ApiController {
     }
 
     def getAppContext = {
+        ActionMenuItem localizationMode;
+        if (session.useDebugLocale) {
+            localizationMode = new ActionMenuItem('Disable localization mode', '/openboxes/images/icons/silk/bug_delete.png', '/openboxes/user/disableLocalizationMode')
+        }
+        else localizationMode = new ActionMenuItem('Enable localization mode', '/openboxes/images/icons/flags/png/ht.png', '/openboxes/user/enableLocalizationMode') 
+
+        List<ActionMenuItem> actionMenuItems = [
+            new ActionMenuItem('Edit Profile', '/openboxes/images/icons/silk/user.png', '/openboxes/user/edit'),
+            localizationMode,
+            new ActionMenuItem('Refresh caches', '/openboxes/images/icons/silk/database_wrench.png', '/openboxes/dashboard/flushCache'),
+            new ActionMenuItem('Logout', '/openboxes/images/icons/silk/door.png', '/openboxes/auth/logout')
+        ]
+
+        for (int i=0; i<actionMenuItems.size(); i++) {
```

Comment:
I guess you could use eachWithIndex to be more "groovy", but I don't really like the ActionMenuItem concept tbh (I'd keep it in obj in config or define it here as list of maps).